src/senti_classifier/senti_classifier.py

```diff
@@ -175,7 +175,7 @@ def disambiguateWordSenses(sentence, word):
     bestScore = 0.0
     result = None
     for synset in wordsynsets:
-        for w in nltk.word_tokenize(sentence):
+        for w in tokens:
```

Comment:
Thanks, for the optimizations.
tokens is undefined here.

Reply:
https://github.com/kevincobain2000/sentiment_classifier/pull/16/commits/56b0e58a8f91a100f9bdc4674bb857016667c63a

Reply:
should work now

Reply:
6f85ca5